cdip_connector/core/connector_base.py

```diff
@@ -46,11 +46,11 @@ async def main(self) -> None:
             for idx in range(0, len(integrations), self.concurrency):
 
                 async with ClientSession() as session:
-
-                    self.logger.info(f"CLIENT_ID: {cdip_settings.KEYCLOAK_CLIENT_ID}")
+                    
+                    self.logger.info(f"Running Integrations for client_id: {cdip_settings.KEYCLOAK_CLIENT_ID}")
 
                     tasks = [
-                        asyncio.ensure_future(self.extract_load(session, integration))
+                        asyncio.ensure_future(self.__class__().extract_load(session, integration))
```

Comment:
@vgarcia13 Here is where I fixed my error -- I create a new instance of the connector for each task.
I can think of many ways to make this better, but for now this is a useful fix.